collectors/python.d.plugin/redis/redis.chart.py

```diff
@@ -189,23 +189,38 @@ def get_data(self):
         :return: dict
         """
         data = self.get_raw_and_parse()
-
         if not data:
             return None
 
+        self.calc_hit_rate(data)
+        self.calc_redis_keys(data)
+        self.calc_redis_rdb_save_operations(data)
+        return data
+
+    @staticmethod
+    def calc_hit_rate(data):
         try:
-            data['hit_rate'] = (
-                    (int(data['keyspace_hits']) * 100) / (int(data['keyspace_hits']) + int(data['keyspace_misses']))
-            )
+            hits = int(data['keyspace_hits'])
+            misses = int(data['keyspace_misses'])
+            data['hit_rate'] = hits * 100 / (hits + misses)
```

Comment:
Are you sure we do not have any possibility of both `hits` and `misses` to be zero?

Reply:
> except (KeyError, ZeroDivisionError)
>  ZeroDivisionError

Reply:
Interesting logic, thank you!